src/rust/ensogl/lib/components/src/drop_down_menu.rs

```diff
@@ -74,7 +76,7 @@ pub mod chooser_hover_area {
 
 ensogl_core::define_endpoints! {
     Input {
-        set_entries         (list_view::entry::AnyModelProvider),
+        set_entries         (Rc<Vec<String>>),
```

Comment:
Would it suffice to use a type alias for `RC<Vec<String>>`?

Reply:
No, because the type checker when infering types would expand that and will hint you with a messy type instead of the right one. Such things should be defined with new types, the previous implementation used a very nice design. This needs to be reverted.